proxy/src/http/source.rs

```diff
@@ -19,6 +19,7 @@ pub fn routes<R>(
     peer: Arc<Mutex<coco::PeerApi>>,
     registry: http::Shared<R>,
     store: Arc<RwLock<kv::Store>>,
+    owner: &http::Shared<Option<coco::User>>,
```

Comment:
This is weird, we should generally own these so that when it goes out of scope the ref count is dropped and what not. It's rare we have to borrow an `Arc`.

Reply:
The other thing is that I generally placed the `owner` after the `peer` because they're closely related.

Reply:
This got resolved by merging: https://github.com/radicle-dev/radicle-upstream/pull/639.